opsdroid/connector/tests/test_connector_rocketchat.py

```diff
@@ -0,0 +1,295 @@
+"""Tests for the RocketChat connector."""
+import asyncio
+import pytest
+import contextlib
+import asynctest.mock as amock
+
+from opsdroid.core import OpsDroid
+from opsdroid.connector.rocketchat import RocketChat
+from opsdroid.events import Message
+from opsdroid.cli.start import configure_lang
+
+configure_lang({})
+
+
+def test_init():
+    """Test that the connector is initialised properly."""
+    connector = RocketChat(
+        {
+            "name": "rocket.chat",
+            "token": "test",
+            "user-id": "userID",
+            "update-interval": 0.1,
+        },
+        opsdroid=OpsDroid(),
+    )
+    assert connector.default_target == "general"
+    assert connector.name == "rocket.chat"
+
+
+def test_missing_token(caplog):
+    """Test that attempt to connect without info raises an error."""
+
+    RocketChat({})
+    assert "Unable to login: Access token is missing." in caplog.text
+
+
+def setUp():
```

Comment:
Do we need the setup? In unittest we have to use this to make sure that every test uses whatever we add inside setUp, but not sure about pyttest - that's why I'm asking 😄